plugins/hashsum_download/web_client/templates/hashsum_download_fileInfo.jade

```diff
@@ -0,0 +1,6 @@
+if file.has('sha512')
+  .g-file-info-line(property="sha512")
+    i.icon-barcode
+    span SHA-512:
+    span.g-bold-part
+      a(href=keyfileUrl(file.id, 'sha512'))  #{abbreviate(file.get('sha512'))}
```

Comment:
Could we have an easy way for a user to select this hash value (for example, to copy to the clipboard) without having to download the file?